### DISCUSSION TITLE
LazyVim keymaps inside JetBrains IdeaVim

### DISCUSSION THREAD

**Discussion by leaf541** [LazyVim/LazyVim#2081] [General]
Hi, so I'm using Webstorm as my default IDE and started trying to incorporate vim to my workflow. I've setup NeoVim but as I'm still not so comfortable with the whole workflow I'm switching between them. What I wanted to ask is it possible to implement the same keybindings inside IdeaVim as LazyVim uses? This would maybe make the transition more comfortable. Also if someone also had this issue I would like to hear your thoughts and ideas.

**Comment by dpetka2001** [comment]
You can change the keybindings to whatever you want. You mean change the keybindings of LazyVim to be the same with what you use in JetBrains, right? Or the opposite?

**Reply by leaf541** [reply]
I was thinking of opposite, use LazyVim keybindings inside IdeaVim. Only thing is I don't know how to achieve that.

**Reply by dpetka2001** [reply]
No idea, never used Idea myself. Maybe google will be more helpful.

**Comment by deifyed** [comment]
It is possible, but someone has to do the heavy lifting. Would love it if anyone has a ideavim config to share. Maybe a good way of listing/exoprting all relevant lazyvim keybinds?

**Comment by jmlucjav** [comment]
see https://www.reddit.com/r/neovim/comments/1giysv5/lazyvim_mappings_for_jetbrains_ides/

this guy is doing a great job https://gist.github.com/mikeslattery/d2f2562e5bbaa7ef036cf9f5a13deff5 eventually they made a repo: https://github.com/cufarvid/lazy-idea
